lint-javascript.yml

```diff
@@ -0,0 +1,19 @@
+---
+include:
+  - remote: "https://raw.githubusercontent.com/jobtome-labs/ci-templates/v3.8.0/templates/lint.yml"
+
+lint:eslint:
+  extends: .lint
+
+  image: "${CI_REGISTRY_IMAGE}/${APP_ESLINT_IMAGE_NAME}:${CI_COMMIT_SHORT_SHA}"
+
+  script: 
+    - cd /app/src && pnpm run eslint-check
```

Comment:
Since it's a template, I think we should make more abstract this command.
What do you think if we just set here the command: `pnpm run <eslint-check-command>` and we specify the command: `cd /app/src && pnpm run eslint-check` as script in `app/package.json` ?

Reply:
mmm the problem is that the path depends on the `Docker workdir`, so having it inside the `package.json` is a problem because the `docker workdir` is different from the `local machine workdir` (we cannot run the lint inside docker in local) so the only way to do this could be create a `docker:eslint-check` command.

what do you think? is a good idea ?

Reply:
Ok, makes sense.